todo_app/app.py

```diff
@@ -1,70 +1,92 @@
 
 from todo_app.data.trello_board import *
-from todo_app.data.task import *
+from todo_app.data.viewmodel import *
 from flask import Flask, render_template, request, redirect
 
-from todo_app.flask_config import Config
 
-app = Flask(__name__)
-app.config.from_object(Config)
+def create_app(board):
```

Comment:
Passing through this board object works nicely for the tests, but breaks running your server locally. Instead, I'd suggest that you create the board inside this function (e.g. line 10) using the environment variables that have been set elsewhere (e.g. in your tests)

You'll then need to change your end-to-end tests, where you create a trello board, to override the environment variables with the new board id, rather than setting it on the board object that is passed through.